### PR TITLE
v1.6.0 recenter_tc - VERSION RELEASE - standardize README, update YAML metadata

### PR DESCRIPTION
**Pull request by mindyls**
# Related Issues
required to close NRLMMD-GEOIPS/geoips#51

# Reviewer Instructions
* Please confirm most recent completed pull request is labeled "DEVELOPMENTAL UPDATE" or "VERSION RELEASE":
    * https://github.com/NRLMMD-GEOIPS/recenter_tc/pulls?q=is%3Apr+is%3Aclosed
* Review "Files changed" tab
* Confirm appropriate testing was completed for these updates.

# Summary
Changes for version 1.6.0 update on repo recenter_tc.
* standardize README and installation
* Update YAML metadata with storm_start_datetime
* Modify test scripts for updated ASCAT test data paths.

See NRLMMD-GEOIPS/geoips#51 for additional repositories included in this update.

# Testing Instructions

Install GEOIPS and clone v1.6.0-release branch for:
* recenter_tc

Tests should return 0:
$GEOIPS_PACKAGES_DIR/recenter_tc/tests/test_all.sh

# Output

# Post Merge Steps
Once this PR has been merged, v1.6.0 will be tagged/released on the recenter_tc main branch.

### CODE REVIEW

File: README.md
Comment (cpcamacho) [inline README.md:18]: @mindyls Should we be using a relative path here?

Comment (mindyls) [reply README.md:18]: @cpcamacho This is a link to another repo, so I don't think relative links will work.  I'm going to resolve, feel free to merge this repo once approved and you are happy with the state of the PR.

**Review by cpcamacho — APPROVED** [review]

**Pull request merged**
